code-orange/MainViewController.swift

```diff
@@ -140,20 +142,22 @@ class MainViewController: UIViewController {
     super.viewDidLoad()
     view.addSubview(mainStack)
     mainStack.pin(to: view, anchors: [.leading(0), .trailing(0), .top(28), .bottom(0)])
-    loadLocations()
+    locations = locationsProvider?.getLocations()
```

Comment:
lets make this into a func "getFreshLocations" 
I think it will be a really useful function

Reply:
agreed!